picard/ui/options/plugins.py

```diff
@@ -86,6 +86,8 @@ def __init__(self, parent=None):
         self.ui.plugins.mimeTypes = self.mimeTypes
         self.ui.plugins.dropEvent = self.dropEvent
         self.ui.plugins.dragEnterEvent = self.dragEnterEvent
+        # hide column initially, shows only when update is availabe
+        self.ui.plugins.hideColumn(3)
```

Comment:
Can you please use a `CONSTANT` to store this `3` ? We don't want to search/replace `3` in the code if we insert a column before this one.